docs/src/examples/modules/Search/Types/index.js

```diff
@@ -21,7 +21,7 @@ const SearchTypesExamples = () => (
       examplePath='modules/Search/Types/SearchExampleCategory'
     />
     <ComponentExample
-      title='Category (Custom Render)'
+      title='Category (Custom Render) v2'
```

Comment:
Let's not version the example titles in this way.  It will be confusing with the semver already in place.